test/test.g.dart

```diff
@@ -1270,37 +1293,557 @@ class Player extends _Player with RealmEntity, RealmObject {
+
+  @override
+  DateTime? get nullableDateProp =>
+      RealmObjectBase.get<DateTime>(this, 'nullableDateProp') as DateTime?;
+  @override
+  set nullableDateProp(DateTime? value) =>
+      RealmObjectBase.set(this, 'nullableDateProp', value);
+
+  @override
+  double? get nullableDoubleProp =>
+      RealmObjectBase.get<double>(this, 'nullableDoubleProp') as double?;
+  @override
+  set nullableDoubleProp(double? value) =>
+      RealmObjectBase.set(this, 'nullableDoubleProp', value);
+
+  @override
+  ObjectId? get nullableObjectIdProp =>
+      RealmObjectBase.get<ObjectId>(this, 'nullableObjectIdProp') as ObjectId?;
+  @override
+  set nullableObjectIdProp(ObjectId? value) =>
+      RealmObjectBase.set(this, 'nullableObjectIdProp', value);
+
+  @override
+  Uuid? get nullableUuidProp =>
+      RealmObjectBase.get<Uuid>(this, 'nullableUuidProp') as Uuid?;
+  @override
+  set nullableUuidProp(Uuid? value) =>
+      RealmObjectBase.set(this, 'nullableUuidProp', value);
+
+  @override
+  int? get nullableIntProp =>
+      RealmObjectBase.get<int>(this, 'nullableIntProp') as int?;
+  @override
+  set nullableIntProp(int? value) =>
+      RealmObjectBase.set(this, 'nullableIntProp', value);
+
+  @override
+  RealmList<String> get strings =>
+      RealmObjectBase.get<String>(this, 'strings') as RealmList<String>;
+  @override
+  set strings(covariant RealmList<String> value) =>
+      throw RealmUnsupportedSetError();
+
+  @override
+  RealmList<bool> get bools =>
+      RealmObjectBase.get<bool>(this, 'bools') as RealmList<bool>;
+  @override
+  set bools(covariant RealmList<bool> value) =>
+      throw RealmUnsupportedSetError();
+
+  @override
+  RealmList<DateTime> get dates =>
+      RealmObjectBase.get<DateTime>(this, 'dates') as RealmList<DateTime>;
+  @override
+  set dates(covariant RealmList<DateTime> value) =>
+      throw RealmUnsupportedSetError();
+
+  @override
+  RealmList<double> get doubles =>
+      RealmObjectBase.get<double>(this, 'doubles') as RealmList<double>;
+  @override
+  set doubles(covariant RealmList<double> value) =>
+      throw RealmUnsupportedSetError();
+
+  @override
+  RealmList<ObjectId> get objectIds =>
+      RealmObjectBase.get<ObjectId>(this, 'objectIds') as RealmList<ObjectId>;
+  @override
+  set objectIds(covariant RealmList<ObjectId> value) =>
+      throw RealmUnsupportedSetError();
+
+  @override
+  RealmList<Uuid> get uuids =>
+      RealmObjectBase.get<Uuid>(this, 'uuids') as RealmList<Uuid>;
+  @override
+  set uuids(covariant RealmList<Uuid> value) =>
+      throw RealmUnsupportedSetError();
+
+  @override
+  RealmList<int> get ints =>
+      RealmObjectBase.get<int>(this, 'ints') as RealmList<int>;
+  @override
+  set ints(covariant RealmList<int> value) => throw RealmUnsupportedSetError();
+
+  @override
+  Stream<RealmObjectChanges<AllTypesEmbedded>> get changes =>
+      RealmObjectBase.getChanges<AllTypesEmbedded>(this);
+
+  @override
+  AllTypesEmbedded freeze() =>
```

Comment:
We can add a test to freeze one embeded object from a collection that belongs to a live top-level object.